components/NavigationSliceZone.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <div class="navigation-slice-zone__wrapper">
+    <div class="navigation-slice-zone">
```

Comment:
Selon BEM, il faudrait plutôt 
```suggestion
  <div class="navigation-slice-zone">
    <div class="navigation-slice-zone__wrapper">
```